Tests/OktaAnalyticsTests/FirebaseAnalyticsProviderTests.swift

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import XCTest
+@testable import OktaAnalytics
+import OktaLogger
+
+final class FirebaseAnalyticsProviderTests: XCTestCase {
+
+    func testTrackEvent_verifyPropertiesWithMaxCharLimit() {
+
+        let properties = ["prop1": "value",
+                          "prop2": "valuevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevaluevalue"
+                            ]
+        let trimmedProperties = FirebaseAnalyticsProvider.trimProperties(properties: properties)
```

Comment:
Please add assertion that length of value for property `prop2` is indeed more than 100